### DISCUSSION TITLE
Добавить отображение количества треков в плейлисте

### DISCUSSION THREAD

**Discussion by F000x** [Chimildic/YaMuTools#65] [Идея] [answered]
В веб-версии Я.Музыка не показывает, сколько треков в плейлисте. Было бы очень полезно сделать отображение кол-ва треков или функцию показать кол-во треков в плейлисте. 

В Я.Музыке плейлист ограничен 10000 треков, поэтому приходится следить, сколько треков сейчас в плейлисте, чтобы иметь потом возможность соединить этот плейлист с другими. Или собираешь плейлист из других листов и думаешь, сколько же там сейчас треков и сколько ещё плейлистов туда влезет. С учетом того, что расширение уже умеет считать кол-во треков (после удаления дубликатов эта информация показывается), думаю будет не очень сложно вынести это в отдельную функцию.

**Comment by kisoid** [comment]
Поддерживаю. Сам столкнулся недавно с этим ограничением. Самое хреновое, что когда добавляешь треки в заполненный плейлист, они какбЭ добавляются, т.е. ни ошибок, ни предупреждений не возникает... Но при этом суммарная длительность плейлиста в часах/минутах перестаёт увеличиваться.

**Comment by Chimildic** [comment, marked as answer]
Добавлять числа к каждому треку или в меню выбора плейлистов не стал. Много времени надо на это. Но добавил общую приписку к описанию плейлиста. Ожидайте в следующем обновлении.
